examples/Extensions/AddAffiliateLocationExtensions.cs

```diff
@@ -0,0 +1,439 @@
+    /// and associates the feed with a campaign for serving affiliate location extensions.
+    /// </summary>
+    public class AddAffiliateLocationExtensions : ExampleBase
+    {
+        // The maximum number of attempts to make to retrieve the FeedMappings before throwing an
+        // exception.
+        private const int MAX_FEEDMAPPING_RETRIEVAL_ATTEMPTS = 10;
+
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            AddAffiliateLocationExtensions codeExample =
+                new AddAffiliateLocationExtensions();
+            Console.WriteLine(codeExample.Description);
+
+            GoogleAdsClient client = new GoogleAdsClient();
+
+            // The customer ID for which the call is made.
+            int customerId = int.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // The retail chain ID. See https://developers.google.com/adwords/api/docs/appendix/codes-formats#chain-ids
+            // for a complete list of valid retail chain IDs.
+            int chainId = int.Parse("INSERT_CHAIN_ID_HERE");
+
+            // The campaign ID for which the affiliate location extensions are added.
+            int campaignId = int.Parse("INSERT_CAMPAIGN_ID_HERE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId, chainId, campaignId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example adds a feed that syncs retail addresses for a given " +
+                    "retail chain ID and associates the feed with a campaign for serving " +
+                    "affiliate location extensions.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The customer ID for which the call is made.</param>
+        /// <param name="chainId">The retail chain ID.</param>
+        /// <param name="campaignId">The campaign ID for which the affiliate location extensions
+        /// are added.</param>
+        public void Run(GoogleAdsClient client, long customerId, long chainId, long campaignId)
+        {
+            try
+            {
+                string aleFeedResourceName = CreateALEFeed(client, customerId, chainId);
+                // After the completion of the Feed ADD operation above the added feed will not be
+                // available for usage in a CampaignFeed until the FeedMappings are created.
+                // We will wait with an exponential back-off policy until the feedmappings have
```

Comment:
feedmappings should either be 'feed mappings' or 'FeedMappings' depending on whether you want to reference the object or the concept.